tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "module": "CommonJS",
-        "lib": ["ES2018"],
+        "lib": ["ES2018", "dom"],
```

Comment:
Hoping to avoid needing to do this... probably time typedoc started using project references. The themes have dom, but not typedoc itself.

Reply:
Does it make sense to publish the themes as separate packages with a simple monorepo setup?  It would make the themes more clearly look like how a third-party theme might be implemented.

Reply:
I'm in favour of this, especially with regards to the long term goals discussed on Discord to make theming easier.